invoker/function-maven-plugin/pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>com.google.cloud.functions.invoker</groupId>
     <artifactId>java-function-invoker-parent</artifactId>
-    <version>1.0.3-SNAPSHOT</version>
+    <version>1.1.0-SNAPSHOT</version>
   </parent>
 
   <groupId>com.google.cloud.functions</groupId>
```

Comment:
@eamonnmcmanus should I have also bumped the version in this pom.xml as well?

Reply:
I'm not sure why you are changing any of the version numbers. Usually we only do that as part of the release process: we would make a branch where the version changes from 1.0.3-SNAPSHOT to 1.0.3, release those artifacts, and then change the version in the main branch to 1.1.0-SNAPSHOT or whatever. (That's all handled by `mvn release:prepare` and `mvn release:perform`.) Besides that, we don't change the version in intermediate changes like this one. If you want to make it so the next release is 1.1.0 rather than 1.0.3 then I would suggest making a PR that just changes the version numbers to 1.1.0-SNAPSHOT and does nothing else.

Reply:
OK, I am not super familiar with maven so I wasn't sure how this worked. I will remove the version number changes from this PR and work our the release process separately.